File: tests/openmp_examples/tasking/task_dep.4b.f90
!$omp   parallel private(i,v) shared(R)
!$omp   single
!$omp       task depend(inout: R)
!$omp       end task
!$omp     task depend(in: R)
!$omp     end task
!$omp     taskwait
!$omp       task depend(inoutset: R)
!$omp         atomic
!$omp       end task
!$omp     task depend(in: R)
!$omp     end task
!$omp   end single
!$omp   end parallel

File: tests/openmp_examples/tasking/task_dep.4b.c
#pragma omp parallel private(i,v) shared(R)
#pragma omp single
#pragma omp task depend(inout: R)
#pragma omp task depend(in: R)
#pragma omp taskwait
#pragma omp task depend(inoutset: R)
#pragma omp atomic
#pragma omp task depend(in: R)
